GDJS/package.json

```diff
@@ -19,6 +20,7 @@
     "prettier": "2.1.2",
     "recursive-readdir": "^2.2.2",
     "shelljs": "^0.8.4",
+    "three": "0.151.2",
```

Comment:
just to check, why are we installing three if it's also embedded in the pixi-runtime/renderers?

Reply:
@D8H so you have an answer to this question? I can't remember why we added it